rest/rest-sqs-testing-amazon-java-sdk/src/test/scala/org/elasticmq/rest/sqs/AmazonJavaSdkTestSuite.scala

```diff
@@ -894,6 +895,26 @@ class AmazonJavaSdkTestSuite extends FunSuite with Matchers with BeforeAndAfter
     msgIdSent should be(msgIdReceived)
   }
 
+  test("should create queue with redrive policy.") {
+    // When
+    val queueUrl = client.createQueue(new CreateQueueRequest("testQueue1")
+      .withAttributes(Map(
+        defaultVisibilityTimeoutAttribute -> "1",
+        redrivePolicyAttribute ->
+          """
+            |{
+            |  "queueName":"dlq1",
```

Comment:
sorry, yes, all params has correct names according docs, will update new ones.

about `defaultVisibilityTimeout` both cases are valid. it can be updated at any time.